data/migrations/20200209204020_create-tables.js

```diff
@@ -0,0 +1,43 @@
+
+exports.up = function(knex) {
+    return knex.schema
+    .createTable('projects', tbl => {
+        tbl.increments('id')
+        tbl.string('project_name', 128)
+            .notNullable()
+        tbl.text('description', 500)
+        tbl.boolean('completed');
```

Comment:
a boolean that indicates if the project has been completed. This column cannot be NULL, the default value should be false.